src/Containers/containerize/ContainerizeCommand.cs

```diff
@@ -185,6 +185,18 @@ internal class ContainerizeCommand : CliRootCommand
 
     internal CliOption<string> ContainerUserOption { get; } = new("--container-user") { Description = "User to run the container as." };
 
+    internal CliOption<bool> GenerateLabelsOption { get; } = new("--generate-labels")
+    {
+        Description = "If true, the tooling may create labels on the generated images.",
+        Arity = ArgumentArity.Zero
+    };
+
+    internal CliOption<bool> GenerateDigestLabelOption { get; } = new("--generate-digest-label")
+    {
+        Description = "If true, the tooling will generate an 'org.opencontainers.image.base.digest' label on the generated images containing the digest of the chosen base image.",
```

Comment:
As with the task, should this check against `GenerateLabelsOption`?

Reply:
Same response: https://github.com/dotnet/sdk/pull/39160#discussion_r1512976442